src/osd/PeeringState.cc

```diff
@@ -6227,6 +6227,13 @@ boost::statechart::result PeeringState::Active::react(const AllReplicasActivated
     pl->send_pg_created(pgid);
   }
 
+  psdout(1) << __func__ << " AllReplicasActivated : Peering->Activating->Active over"
```

Comment:
psdout(10) I think

Reply:
psdout(1) is to correspond to the log level of PeeringState::start_peering_interval()

Reply:
Hrm, I'm inclined to raise both to 10 then.  @neha-ojha @rzarzynski Do we want any peering messages at debug 1?

Reply:
I have found some of them useful to debug things at default log level, so I don't mind this addition.